frontend-ssr/components/lessons/LessonComments/LessonComments.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { connect, ConnectedProps } from 'react-redux';
+import { RootState } from '~/redux/root.reducer';
+
+import {
+  loadComments,
+} from '../../../redux/lesson-comments/lesson-comments.actions';
```

Comment:
You can import them using `~` alias which should result in a smaller path. Same for the `LessonCommentsService`.

It's not a big deal but if we ever move this component, we will not have to re-write all the paths.